test/cypress/integration/SearchUserPage.spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Search User Page', () => {
+	beforeEach(() => {
+		cy.visit('/search-users')
+		cy.injectAxe()
+	})
+
+	it('should load page', () => {
+		cy.get('h1').should('contain', 'Search Users Page')
```

Comment:
Same comment as other spec file

```suggestion
		cy.get('h4').should('contain', 'Search Users Page')
```